tests/php/Core/Application/Security/ProviderConfiguration/OpenId/UseCase/UpdateOpenIdConfiguration/UpdateOpenIdConfigurationTest.php

```diff
@@ -36,98 +36,101 @@
-        $request->isActive = true;
-        $request->isForced = true;
-        $request->trustedClientAddresses = [];
-        $request->blacklistClientAddresses = [];
-        $request->baseUrl = 'http://127.0.0.1/auth/openid-connect';
-        $request->authorizationEndpoint = '/authorization';
-        $request->tokenEndpoint = '/token';
-        $request->introspectionTokenEndpoint = '/introspect';
-        $request->userInformationEndpoint = '/userinfo';
-        $request->endSessionEndpoint = '/logout';
-        $request->connectionScopes = [];
-        $request->loginClaim = 'preferred_username';
-        $request->clientId = 'MyCl1ientId';
-        $request->clientSecret = 'MyCl1ientSuperSecr3tKey';
-        $request->authenticationType = 'client_secret_post';
-        $request->verifyPeer = false;
-
-        $openIdConfiguration = OpenIdConfigurationFactory::createFromRequest($request);
-
-        $this->repository
-            ->expects($this->once())
-            ->method('updateConfiguration')
-            ->with($openIdConfiguration);
-
-        $this->presenter
-            ->expects($this->once())
-            ->method('setResponseStatus')
-            ->with(new NoContentResponse());
-
-        $useCase = new UpdateOpenIdConfiguration($this->repository);
-        $useCase($this->presenter, $request);
-    }
-
-    /**
-     * Test that the useCase is correctly executed with correct parameters
-     *
-     * @return void
-     */
-    public function testUseCaseWithInvalidParameters(): void
-    {
-        $request = new UpdateOpenIdConfigurationRequest();
-        $request->isActive = true;
-        $request->isForced = true;
-        $request->trustedClientAddresses = ["abcd_.@"];
-        $request->blacklistClientAddresses = [];
-        $request->baseUrl = 'http://127.0.0.1/auth/openid-connect';
-        $request->authorizationEndpoint = '/authorization';
-        $request->tokenEndpoint = '/token';
-        $request->introspectionTokenEndpoint = '/introspect';
-        $request->userInformationEndpoint = '/userinfo';
-        $request->endSessionEndpoint = '/logout';
-        $request->connectionScopes = [];
-        $request->loginClaim = 'preferred_username';
-        $request->clientId = 'MyCl1ientId';
-        $request->clientSecret = 'MyCl1ientSuperSecr3tKey';
-        $request->authenticationType = 'client_secret_post';
-        $request->verifyPeer = false;
-
-        $this->presenter
-            ->expects($this->once())
-            ->method('setResponseStatus')
-            ->with(new ErrorResponse(
-                '[OpenIdConfiguration::trustedClientAddresses] The value "abcd_.@" '
-                . 'was expected to be a valid ip address or domain name'
-            ));
-
-        $useCase = new UpdateOpenIdConfiguration($this->repository);
-
-        $useCase($this->presenter, $request);
-    }
+    //@todo: Reimplement those tests while handling update openid configuration extension.
```

Comment:
use `->skip('Reimplement those tests while handling update openid configuration extension');` instead of comment tests